draft-ietf-taps-interface.md

```diff
@@ -2327,10 +2327,14 @@ Type:
 
 This property, if applicable, represents the maximum Message size that can be
 sent without incurring network-layer fragmentation at the sender.
-It is specified as a number of bytes. It exposes a value to the application
+It is specified as a number of bytes and is less than or equal to the
+Maximum Message Size on Send.
+It exposes a readable value to the application
 based on the Maximum Packet Size (MPS) as described in Datagram PLPMTUD {{?RFC8899}}.
```

Comment:
To address #1349, I'd suggest the following change (added sentence), here:
```suggestion
based on the Maximum Packet Size (MPS) as described in Datagram PLPMTUD {{?RFC8899}}. This means that the value of this property can change over time (it can be different before and after executing Datagram PLPMTUD).
```

Reply:
What does execute DPLPMTUD mean? Is this crisper?

/as described in Datagram PLPMTUD {{?RFC8899}}./. The value of this property can change over time (and can be updated by Datagram PLPMTUD  {{?RFC8899}})./